components/library/botan/botan.p5m

```diff
@@ -14,293 +14,397 @@
+file path=usr/include/botan-2/botan/sym_algo.h
+file path=usr/include/botan-2/botan/symkey.h
+file path=usr/include/botan-2/botan/system_rng.h
+file path=usr/include/botan-2/botan/threefish.h
+file path=usr/include/botan-2/botan/threefish_512.h
+file path=usr/include/botan-2/botan/tiger.h
+file path=usr/include/botan-2/botan/tls_alert.h
+file path=usr/include/botan-2/botan/tls_algos.h
+file path=usr/include/botan-2/botan/tls_blocking.h
+file path=usr/include/botan-2/botan/tls_callbacks.h
+file path=usr/include/botan-2/botan/tls_channel.h
+file path=usr/include/botan-2/botan/tls_ciphersuite.h
+file path=usr/include/botan-2/botan/tls_client.h
+file path=usr/include/botan-2/botan/tls_exceptn.h
+file path=usr/include/botan-2/botan/tls_extensions.h
+file path=usr/include/botan-2/botan/tls_handshake_msg.h
+file path=usr/include/botan-2/botan/tls_magic.h
+file path=usr/include/botan-2/botan/tls_messages.h
+file path=usr/include/botan-2/botan/tls_policy.h
+file path=usr/include/botan-2/botan/tls_server.h
+file path=usr/include/botan-2/botan/tls_server_info.h
+file path=usr/include/botan-2/botan/tls_session.h
+file path=usr/include/botan-2/botan/tls_session_manager.h
+file path=usr/include/botan-2/botan/tls_session_manager_sql.h
+file path=usr/include/botan-2/botan/tls_version.h
+file path=usr/include/botan-2/botan/totp.h
+file path=usr/include/botan-2/botan/tss.h
+file path=usr/include/botan-2/botan/twofish.h
+file path=usr/include/botan-2/botan/types.h
+file path=usr/include/botan-2/botan/uuid.h
+file path=usr/include/botan-2/botan/version.h
+file path=usr/include/botan-2/botan/whrlpool.h
+file path=usr/include/botan-2/botan/workfactor.h
+file path=usr/include/botan-2/botan/x509_ca.h
+file path=usr/include/botan-2/botan/x509_crl.h
+file path=usr/include/botan-2/botan/x509_dn.h
+file path=usr/include/botan-2/botan/x509_ext.h
+file path=usr/include/botan-2/botan/x509_key.h
+file path=usr/include/botan-2/botan/x509_obj.h
+file path=usr/include/botan-2/botan/x509cert.h
+file path=usr/include/botan-2/botan/x509path.h
+file path=usr/include/botan-2/botan/x509self.h
+file path=usr/include/botan-2/botan/x919_mac.h
+file path=usr/include/botan-2/botan/xmss.h
+file path=usr/include/botan-2/botan/xmss_hash.h
+file path=usr/include/botan-2/botan/xmss_key_pair.h
+file path=usr/include/botan-2/botan/xmss_parameters.h
+file path=usr/include/botan-2/botan/xmss_privatekey.h
+file path=usr/include/botan-2/botan/xmss_publickey.h
+file path=usr/include/botan-2/botan/xmss_wots.h
+file path=usr/include/botan-2/botan/xmss_wots_parameters.h
+file path=usr/include/botan-2/botan/xmss_wots_privatekey.h
+file path=usr/include/botan-2/botan/xmss_wots_publickey.h
+file path=usr/include/botan-2/botan/xtea.h
+file path=usr/include/botan-2/botan/xts.h
+file path=usr/include/botan-2/botan/zfec.h
+file path=usr/include/botan-2/botan/zlib.h
+file path=usr/lib/$(MACH64)/libbotan-2.a
+link path=usr/lib/$(MACH64)/libbotan-2.so target=libbotan-2.so.19.19.2
```

Comment:
This will fail to publish on the build server.

Reply:
Why will it fail?

Reply:
You are right, it won't fail. It would devliver an unwanted static lib. BTW: I have created #9242 which will make dropping static libs optional after some discussions.